asyncapi-2025-goal.md

```diff
@@ -0,0 +1,118 @@
+Carrying on from Thulie, it's time to have an open discussion on the goals for building and maintaining the AsyncAPI community. The Community Manager needs to ensure the goals are addressed and that each program/initiative has a dedicated person responsible for it.
+
+Things I think we should focus on in 2025:
+
+- Increase Maintainers
+- Onboarding Docs and Video Content
+- Community Working Group
```

Comment:
remember this list is still a mix of objectives and action points

- Increase number of maintainers and assure their retention (not sure grammarly correct) 

You see here is in a nice objective - and someone also can quickly understand what problem it solves

- Onboarding materials
- Community WG

these are action points already - solutions, like introduction of grant for maintainers.

so community first need to see what problems are there and objectives to solve them.

then you suggest possible solutions - creation of onboarding materials - but others might have better suggestions

Reply:
Thanks, @derberg. I applied your suggestions. When you get time, let me know how it looks before I submit it to the community next week after the conference.